test/passing/open-auto.ml.ref

```diff
@@ -31,10 +31,10 @@ let _ =
   y
 
 let () =
-  ( (let open Term in
-    term_result
-      ( const Phases.phase1 $ arch $ hub_id $ build_dir $ logs_dir
-      $ setup_logs ))
+  ( Term.(
+      term_result
+        ( const Phases.phase1 $ arch $ hub_id $ build_dir $ logs_dir
+        $ setup_logs ))
```

Comment:
I like that we get rid of this "feature". I hope this was not used in the wild.

Reply:
Note that using the `let open` form in case the body breaks is entirely intentional. Note in particular that this change does not save any lines and increases the indentation uselessly. So FTR this looks like a regression to me. I agree that the parenthesization in the diff is odd though.

Reply:
I ran test_branch with the wrong arguments and misunderstood the change.
Indeed this seems to remove the whole feature.

Reply:
The thing is that the box layout and the formatting now depends on whether it fits or breaks, in a way that is not feasible with the fits_breaks function, so I need to distinguish the 2 different implementations like I did in this PR

Reply:
The issue with `let-open=auto` is that sometimes it doesn't stabilize, eg:
```ocaml
let rec pp_hpred_env pe0 envo f hpred =
  color_wrapper pe0 f hpred ~f:(fun pe f hpred ->
      match hpred with
      | Hpointsto (e, se, te) ->
          let pe' =
            match (e, se) with
            | Lvar pvar, Eexp (Var _, _) when not (Pvar.is_global pvar) ->
                Pp.{pe with obj_sub= None} (* dont use obj sub on the var defining it *)
            | _ ->
                pe
          in
          fooooooooooooooooo
    )
```